api/azure/api_test.go

```diff
@@ -0,0 +1,38 @@
+// +build small
+
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package azure
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"mime/multipart"
+	"path"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtractFiles(t *testing.T) {
+	_, filename, _, _ := runtime.Caller(0)
+	data, err := ioutil.ReadFile(path.Join(path.Dir(filename), "artifact_test.zip"))
```

Comment:
Perhaps add a comment to explain where is `artifact_test.zip` and what is inside.

Reply:
Done.